### ISSUE TITLE
Compiling osd_gpu_obj with Visual Studio 2010 fails

### ISSUE THREAD

**Issue by foundry-milan** [PixarAnimationStudios/OpenSubdiv#832]
For some reason, CMake fails to add include folder "c:\Program Files (x86)\Windows Kits\8.1\Include\shared\" for Visual Studio 2010 target, for the project "osd_gpu_obj".

Before adding that folder in manually I got the following compilation errors:
"Cannot open include file: 'winapifamily.h'"

The chain of includes is approximately the following: "GL.h" includes require "Windows.h" and it tries to include "Winapifamily.h" and fails.

Let me know if you need more info or if I can do anything else to help resolve this issue.

**Comment by jtran56** [comment]
Filed as internal issue #151672.

**Comment by davidgyu** [comment]
Marking this closed based on recent cross platform testing. We've fixed several build configuration issues related to VS versions and Window Kit versions (e.g. #1173). If you are still blocked by build errors please open a new issue and we'll take a look. Thanks!
